### ISSUE TITLE
Match Loss during training is very large

### ISSUE THREAD

**Issue by manuelli** [RobotLocomotion/pytorch-dense-correspondence#56]
Currently it is somewhere in the vicinity of `0.1` after `3500` steps. This translates to an average descriptor distance between matches off `sqrt(0.1) = 0.3` which seems incredibly large given that `M_pixel = 0.5`. This is doubly confusing since the hard negative rate for masked non-matches is fairly low at 4%. So only 4% of non-matches are below the 0.3 distance.

I think this could potentially be fixed by lowering the learning rate.
